src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardRetryingRequestResolver.java

```diff
@@ -156,14 +155,14 @@ public void update()
         });
 
         //Lets get all keys with 0 residual delay:
-        final Set<IToken> retryables = delays.keySet().stream().filter(t -> delays.get(t) == 0).collect(Collectors.toSet());
-        final Set<IToken> successfully = retryables.stream().filter(t -> {
-            final Set<IToken> blackList = assignedRequests.get(t) < getMaximalTries() ? ImmutableSet.of() : ImmutableSet.of(id);
+        final Set<IToken<?>> retryables = delays.keySet().stream().filter(t -> delays.get(t) == 0).collect(Collectors.toSet());
+        final Set<IToken<?>> successfully = retryables.stream().filter(t -> {
+            final Set<IToken<?>> blackList = assignedRequests.get(t) < getMaximalTries() ? ImmutableSet.of() : ImmutableSet.of(id);
 
             Integer currentAttempt = assignedRequests.get(t);
 
             this.setCurrent(t);
-            final IToken resultingResolver = manager.reassignRequest(t, blackList);
+            final IToken<?> resultingResolver = manager.reassignRequest(t, blackList);
             this.setCurrent(null);
 
             assignedRequests.put(t, ++currentAttempt);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS881)